SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -160,8 +160,13 @@ private void populateTaskEmailProperties(Map<String, Object> templateProperties,
     Optional<String> directory = taskManager.getDirectory(taskId);
 
     templateProperties.put("singularityTaskLink", getSingularityTaskLink(taskId));
-    templateProperties.put("stdout", getTaskLogFile(taskId, "stdout", task, directory).or(new String[0]));
-    templateProperties.put("stderr", getTaskLogFile(taskId, "stderr", task, directory).or(new String[0]));
+
+    Map<String, String[]> logTails = Maps.newHashMap();
+    for (String file : maybeSmtpConfiguration.get().getTaskEmailTailFiles()) {
+      logTails.put(file, getTaskLogFile(taskId, file, task, directory).or(new String[0]));
```

Comment:
one thing i remembered -- in the case where people are using the SingularityExecutor, `service.log` lives inside a folder named after the task ID. could we do a replacement on the `file` variable such that `$MESOS_TASK_ID` becomes the value of `taskId` ?